### PR TITLE
Import chatbot

### PR DESCRIPTION
**Pull request by jhankin**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ gem 'turbolinks',   '5.0.1'
 gem 'jbuilder',     '2.4.1'
 gem 'sidekiq',      '4.1.2'
 gem 'pg',           '0.20.0'
+gem 'pry'
```
Comment (seansu4you87) [inline Gemfile]: let's add this in the `development` and `test` group.  Will prevent us from accidentally leaving a debug statement in production

File: Gemfile [hunk 2]
```diff
@@ -19,6 +20,7 @@ gem 'rack-protection', require: false
 group :development, :test do
   gem 'sqlite3', '1.3.12'
   gem 'byebug',  '9.0.0', platform: :mri
```
Comment (seansu4you87) [inline Gemfile]: If we add pry, we can prolly kill `byebug`

**Comment by seansu4you87** [comment]
my comments are not blocking

File: Gemfile (hunk 2 above)
Comment (jhankin) [inline Gemfile]: I think @khadijah-coakley added `byebug` so if that's her preferred debugger, I'd rather leave it.

**Pull request merged**